mcux/drivers/imx/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ zephyr_library_compile_definitions_ifdef(
   CONFIG_HAS_MCUX_CACHE FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL
 )
 
+zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_12B1MSPS_SAR	fsl_adc.c)
```

Comment:
How come you are using such an odd name? I cannot relate this naming to the i.MX RT10xx datasheets?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/34078#discussion_r613277932

Reply:
@MaureenHelm said to change it according to the MCUXpresso shim drivers convention 
https://github.com/zephyrproject-rtos/zephyr/pull/34078#discussion_r613277932